morpheus.ipynb

```diff
@@ -145,15 +2773,867 @@
+T0u+XwgcR5+9Fc3r+A7dtzzOuOYDezePzjXcROahgWjzs9SASvv7t7Q5WPCDI3Z5e/o+pdmBrOBP6XCPiNZmMi+P/O4pG288xsUL0LJa0aUOpKel0Om0EkkbcFDiYuRKsdDfzcK3d3nzWzd+b4u+TwFuCwrl8F6SG2IO7IjyQu6ltzFnHBtD3wCeC85YwrXWME8EuPO5mvAh8jGq/9iPPjlsCRpfHfRlwwXg6cSvQ8mURcuG4KPG9m2wFvJRpB6+V81s/p++W45xJJw77ERe2BRDLoLCIp8hbiwvZsM3sb0TtgVs5jY6IRdWypXGfmNFOBnYhG1HrATHfvT1x8r0Y0hg4iEkqfJxqDuxM3OD7g0Xvg41XrO59ovK5FXMy/kMPOAbYlkmh/yvHnEY2WZ4lG3RvE+Xsb4iL9JXcfQJzX3040FA7I7TCXSHC8ltvyxJz2YCJZsztxcQ/RAOgHfIFo3KwPbAWMJBoNE3IbnJ3bfxLwZSKhMQLYPOP+fcDXzKwf8AviLvN2REP3C8COROPo+lzuasCxOc5Y4BtEHViP5Z9PVsl1eoNoXLyDSAbsQdSl/xKN6WOI/XoWkTgelNvz62a2GdFwPpqoM4cSDbz5wEXuvjpR564kGoCb5/57kti/H8t1uYa4drmJqKcQdWI1opH0odx3RULhKaLOrwXs4dFj43qi8bUF0aPlWqKxNCa39zVZvjVz+IdyOdPc/d05j77AaDPrT/SM+WFu1yH0Uu7+X2Cxmb2dWOcJRLtlJ+L64mGijswH3pk/z8jJb6GS3PgOUf/+SyQkRgPrAFsT+3lNKom3XYD9iRhSxLetiKQJRAP3zlzOq8TxBFGv/4fo3XEyUVcmuvseWa4Ds66/F/hZ9hzZh6jrTiQF+prZOsCuOe9vZ5lXI25Gr0kcuw8TSZkPEsfdl4FFZjaESNg/kdOeQiR+i2ELc1mnk9d4wKpE3Z5KnH8/RNT3eUTMORX4SD4ydGWW9/O5fi25P64ikiun57Z4V5ZvHeAEYGTG5QtyW79A1NsdiHi4ARG71sl9U1xzQiTBbyOO+/uyTBDxvR8RV/+Z2+sC4tiCOOb7EecOiGTO3ln+7xKx/Xbi3EBu45uJeFrcyFhI7NMZRF2YndunHKu2Bb7i7psVAywehT6GOL5/QiS2IBLZ5Xj46Vzf4pxxsJltlH8PIuLid4CNct2+ltt9gzwv7Jrluo9IRk4n6vv6RN3YxN3PIvb5IRnzppTKb0TCbCCxn75E1NntgD8T5z2I/fOMu7/Esvq6+w5EL6SiV9OXgFl5Tfdt4nzULkoIdSMzW524i3e8u79W7/KUuXtLNhg3BHYws63qXaYqu2ZA/yDxyN3u9S5Qwcz2A2a4+8R6l6UVfYnA+Wt3fxdxMXtifYsky1HuTlo8990P+K2ZPUw0FGo9cjoB+IaZfR0Y5u7ziDvq2wH35gXIXkTDRgTgKXdvzzvZ3geck3Xoj8BaZjaga4smVZ529wfz94lEY2gB0djtQ1xUlx/p+CNxUfkB4iJ1NaKxcB9xcX0PcVE7E7jK3ecQd3gH5/R9crovEhfmrwOPu/t/iLvVLwEXezx2eGH+/W7ijmjRG+Y4osH/i5ynEReur+e8BxINrI2JBtluWb6FwBVE78ZJOe69RINulrtPyfn9wMwmEQ2zScTNrCm5/Lfm9hkAfIu4yP8bsEZO+zBxIf0U8A/ikZgBRCNum9xGuPvfiEbCe4G/EA2xDYhkzWSiV93dxPX0pUSDa32iEURu48XADe7+Ym7jUUSiYiHRU+VKonFdOJc4T68B3JXH3eFEL77NiUZy0aj8PJFMK6afk9eafYCrc5xziYRLYXnnkxaPR2dGZ5ley/12h7ufluMUPcbek2X6upnNIxq4byEaY8+7+71FmYhG5yM5HTnPvYieBTcS9a4/sHZum88QPa6L/TWO6OV8PVGn1iESV9OIOrcOkWxcL8dfx8x+kfO/wt1bSuv4V3dfRNTRRURDdx/i+Blemv5fuY02zPlsQdTdGTnO5fRu44lkUJEQmlD6+y5iH65KJGlWJRrPa5Sm34lI/L2fqCf7EQnMwUSPx6LH4LNU9snDOe2mHq84+D0wNpM47yKSRicS+2ztHHdXogfMVaVyL8rfDfiRmT1EJKqGUknsQCRkjs3xbs3/DSISus8T188nZ2zajaiXg4n4OYRIxK5KJGcGEcnx84mE13VZjj8Rbb+iN8r8PL4WEfFmWM5vU+JYHED0zPk0kSjaP8uykOiR9lqW6wPA57LMf8/1KeJ0C5G4acntcWCu383u/oq7F72FphDHwS+znFuU9t+uRBwmt1eR6P8bkdy7njheJub6XkLs04/kuIfn+H2JxJgTCYuROU3x6FYfIr7/ncrNphbivPBWIjm8epalHKvuyfUoDCF62pxW6tSwIOPhzsD4jFP/IJJjtwK4e9EzdRhxDltM7P8lxPloWyJptzFwp5lNJZJfQ4l9vw2xr7clzkWPAZ8ysy8S54XxWZYnqCS2Ic6ZE4gYOxDYMeP11qX1HJHlraVIwBXXA1DaZ+4+mejJ1i5KCHWTvKvzB+BSd7+2rfHrxd1fJSpfQ72Tyd2fy58ziANvh/qWaCm7APub2TTiQNzTzC6pb5GW8izwbClAXkMELuk5vkrc9dmGuDu3zMuA3f0y4sJhHvAXM9uTuMi5qJRg2tzdT+6+YkuDm1v6fQlRXwr9S78bsEOpHg3NhKN0nwWl31uIi+z+wJfdfRTx/pXyPlvg7tOJuFG8/2BENkT+mvOonvdMIllQ+DnR6H+JuEjeuh0vtVxM5dqypWp+EEmTQeQdenf/N3GxvZC4q74/0XC4jEgIHUpcUB9KXJC/yd2PIZIKxeMfRX1uofKY3GIiafAGccd/8/z/XcQd7j2IpPl+RANtH6JhUO1M4t0WT7r7pu5+fs5ncf5/CdH7chhxvAwtTdvi2Z8/161P/mwBvk/0MjigNP54olFwa+mYG+nuR+a8H6Hyrp+Ls
```

Comment:
Is it your intention to change the notebook? Looks like the dataset is inside there... is this dataset labeled for distribution?

Reply:
Its not :) 
I thought I fixed this with  the 'reverted jupyter notebook to master-version'-commit but seems like I didnt